Tests/BSONTests/BSONCorpusTests.swift

```diff
@@ -98,19 +98,49 @@ final class BSONCorpusTests: BSONTestCase {
 
     // swiftlint:disable:next cyclomatic_complexity
     func testBSONCorpus() throws {
+        let SKIPPED_CORPUS_TESTS = [
+            "Decimal128":
+                [
+                    // TODO: SWIFT-962
+                    "Exact rounding",
+                    "[dqbsr531] negatives (Rounded)",
+                    "[dqbsr431] check rounding modes heeded (Rounded)",
+                    "OK2",
+                    // TODO: SWIFT-965
+                    "[decq438] clamped zeros... (Clamped)",
+                    "[decq418] clamped zeros... (Clamped)"
+                ],
+            "Array":
+                [
```

Comment:
can we add a similar TODO for these? (ticket is SWIFT-963)

Reply:
yep